app/src/androidTest/java/com/duckduckgo/app/browser/httpauth/WebViewHttpAuthStoreTest.kt

```diff
@@ -41,13 +43,19 @@ class WebViewHttpAuthStoreTest {
     val coroutineRule = CoroutineTestRule()
 
     private val context = InstrumentationRegistry.getInstrumentation().targetContext
+    private val webViewDatabaseProvider: Provider<WebViewDatabase> = mock()
```

Comment:
nit: you don't really need a mock here. `Provider` it's a SAM interface so you could just do a lambda.